src/foam/u2/Element.js

```diff
@@ -52,6 +52,66 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
```

Comment:
Is this diff out of date?  There already exists a CSS axiom in Element.js, which looks similar to this except for expandCSS()